src/Nox.Types.EntityFramework/Configurations/NoxDatabaseConfigurator.cs

```diff
@@ -230,8 +233,35 @@ public abstract class NoxDatabaseConfigurator : INoxDatabaseConfigurator
                 databaseConfigurationForForeignKey.ConfigureEntityProperty(codeGeneratorState, builder, foreignEntityKeyDefinition, entity, false);
             }
         }
+        
+        protected virtual IList<IndexBuilder> ConfigureUniqueAttributeConstraints(IEntityBuilder builder, Entity entity)
+        {
```

Comment:
This method is a bit difficult to read. Can we refactor it a bit? We could place the logic int he nested inside dedicated methods for creating audit properties.

Reply:
Desired changes were applied